admin-tools/lambda/riff-raff.yaml

```diff
@@ -0,0 +1,12 @@
+stacks: [media-service]
+regions: [eu-west-1]
+deployments:
+  admin-tools-lambda:
+    type: aws-lambda
+    parameters:
+      bucket: media-service-dist
+      fileName: admin-tools-lambda.jar
+      prefixStack: false
+      functionNames:
+        - "grid-admin-tools-lambda-"
+        - "grid-batch-index-input-generator-lambda-"
```

Comment:
now it is
`admin-tools-image-projection-lambda-{stage}`
`admin-tools-image-batch-index-lambda-{stage}`